gitlistobjectbysize.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+# work over each commit and append all files in tree to $tempFile
+tempFile=$(mktemp)
+IFS=$'\n'
+for commitSHA1 in $(git rev-list --all); do
+	git ls-tree -r --long "$commitSHA1" >>"$tempFile"
+done
+
+# sort files by SHA1, de-dupe list and finally re-sort by filesize
+sort --key 3 "$tempFile" | \
+	uniq | \
+	sort --key 4 --numeric-sort --reverse
+
+# remove temp file
+rm "$tempFile"
```

Comment:
i think you can do `git rev-list --all | xargs git ls-tree -r --long | sort --key 3 | uniq | sort --key 4 --numeric-sort --reverse` (without temp file)

Reply:
Thanks. 
I got that script from [here](https://gist.github.com/magnetikonline/dd5837d597722c9c2d5dfa16d8efe5b9).
I should not have committed it.
I was investigating [this Travis issue](https://travis-ci.org/github/transientskp/pyse/builds/727059272):
```
This repository is over its data quota.
```

So I investigated that using gitlistobjectbysize.sh:

```
100644 blob c946c7699e484795841a65a5a76e8267acbce451 1048576    test/data/SWIFT_554620-130504.image/table.f0_TSM0
100644 blob 442a8965a21b4b3759a13f975a17c30e9b66b0e3   98816    test/data/SWIFT_554620-130504.image/logtable/table.f0
```

Reply:
Can those two files be removed?
Requires rewriting history:
```
git filter-branch \
	--tree-filter "rm -f something/really/large.iso" \
	-- --all
```